example/gen_all.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 set -e
-../target/debug/diplomat-tool js js/ --docs js/docs/ --docs-base-urls=*:https://unicode-org.github.io/icu4x-docs/doc/ 
+../target/debug/diplomat-tool js js/api --docs js/docs/ --docs-base-urls=*:https://unicode-org.github.io/icu4x-docs/doc/ 
```

Comment:
we should eventually:

 - move these all over to cargo-make (or a smarter bash script)
 - make them able to delete existing generated stuff so old files don't stick around

not necessary here though